apps/salga-mobile-api/salga-mobile-api-web.yml

```diff
@@ -0,0 +1,66 @@
+---
+- hosts:
+    - salga-mobile-api
+  become: yes
+  roles:
+    - dokku_bot.ansible_dokku
+
+  vars:
+    instance_name: "salga-mobile-api-{{ env_name }}"
+    app_name: "{{ instance_name }}-web"
+    dokku_daemon_install: false # We install this using a dedicated playbook
+    dokku_plugins:
+      - name: letsencrypt
+        url: https://github.com/dokku/dokku-letsencrypt.git
+
+  tasks:
+    - name: "Dokku app exists"
+      dokku_app:
+        app: "{{ app_name }}"
+
+    - name: "Dokku app config is set"
+      dokku_config:
+        app: "{{ app_name }}"
+        config:
+          APM_SERVER: "http://ip-172-31-33-240.eu-west-1.compute.internal:8200"
+          DB_HOST: "{{ lookup('passwordstore', 'apps/salga-mobile-api/{{ env_name }}/DB subkey=HOST')}}"
```

Comment:
it doesn't make sense to put db host and port into passwordstore - just put that into the playbook. only secrets and the username that accompanies a secret need to be in the secret store.

no need to change here now, but for future.